app/components/LatestDataRefresh/LatestDataRefresh.tsx

```diff
@@ -0,0 +1,51 @@
+import cx from "classnames";
+import { useRouter } from "next/router";
+import { useQuery } from "react-query";
+import { LatestMetadataResponse } from "../../../shared/types/api";
+import { Button } from "../Button";
+import clientFetcher from "../../utilities/clientFetcher";
+import styles from "./LatestDataRefresh.module.scss";
+
+type LatestDataRefreshProps = {
+  dataUpdatedAtTimeUtc: string;
+  className?: string;
+};
+
+const getLatestMetadata = async (): Promise<LatestMetadataResponse> =>
+  clientFetcher("/api/data/latest-metadata");
+
+export const LatestDataRefresh = ({
+  className,
+  dataUpdatedAtTimeUtc,
+}: LatestDataRefreshProps) => {
+  const { data, error } = useQuery("latest-metadata", getLatestMetadata, {
```

Comment:
Should probably do something with [`initialData` config](https://react-query.tanstack.com/guides/initial-query-data#using-initialdata-to-prepopulate-a-query) here so it doesn't do a call on initial mount/render

Reply:
I also went with the more "simpler" approach here https://react-query.tanstack.com/guides/ssr#using-nextjs

Reply:
I think this approach is fine 🙌 No real need to over-optimise it right now. 

Also, I think we'd maybe want to refactor the metadata endpoint (so we can re-use the same logic). Probably best to do that in a separate PR